wallet/res/values/strings-extra.xml

```diff
@@ -67,16 +78,16 @@
     <string name="payments_tab_pay_label">Send</string>
     <string name="payments_tab_receive_label">Receive</string>
     <string name="payments_pay_by_qr_title">Send by</string>
-    <string name="payments_pay_by_qr_sub_title">Scanning QR Code</string>
+    <string name="payments_pay_by_qr_sub_title">Scanning QR code</string>
     <string name="payments_pay_by_qr_button">Scan</string>
     <string name="payments_pay_to_clipboard_title">Send to copied address</string>
     <string name="payments_pay_to_clipboard_sub_title">No address copied</string>
     <string name="payments_pay_to_clipboard_button">Send</string>
 
     <string name="forgot_pin_title">Forgot PIN?</string>
-    <string name="forgot_pin_message">You can set a new PIN with the help of your recovery phrase?</string>
-    <string name="forgot_pin_instruction_1">Enter Recovery Phrase to recover your wallet</string>
-    <string name="forgot_pin_instruction_2">Setup a new PIN</string>
+    <string name="forgot_pin_message">You can set a new PIN with the help of your recovery phrase.</string>
+    <string name="forgot_pin_instruction_1">Enter recovery phrase to recover your wallet</string>
+    <string name="forgot_pin_instruction_2">Set up a new PIN</string>
```

Comment:
The first line is not a question.

Set up is a verp (setup is a noun).

This goes to a screen that the android app doesn't currently use, which would allow the recovery phrase to be used to reset the PIN.  That feature comes later.